middleware/twitchauth.go

```diff
@@ -144,3 +145,63 @@ func TwitchUserInfo(c *fiber.Ctx) error {
 	}
 	return c.Status(fiber.StatusAccepted).JSON(userInfo.Data)
 }
+
+func ModifyBroadcastInformation(c *fiber.Ctx) error {
+	// TODO: take in query data for this.
+	type Query struct {
+		Title string `query:"title"`
+		Game  string `query:"game"`
```

Comment:
figure out a way to search through the categories. for example if a user wanted to change the game, they would type !setgame fort and it would change the game to fornite in this case.